### PR TITLE
docs(:sparkles:): release 5.7.1

### PR DESCRIPTION
**Pull request by afc163**
- 💄 Migrate Component Token of Menu from 4.x less variables. [#43573](https://github.com/ant-design/ant-design/pull/43573) [@li-jia-nan](https://github.com/li-jia-nan)
- 🐞 Fix QRCode throws `Can't resolve 'antd/lib/qr-code'` in Next.js 13. [#43572](https://github.com/ant-design/ant-design/issues/43572)
- 🐞 Fix that antd components usage in Next.js App Router, check the [documentation](/docs/react/use-with-next#using-nextjs-app-router). [#43573](https://github.com/ant-design/ant-design/pull/43573) [@li-jia-nan](https://github.com/li-jia-nan)
- 🐞 Fix Checkbox both set `checked` and `indeterminate` prop will not show as `indeterminate` style. [#43626](https://github.com/ant-design/ant-design/pull/43626)
- 🐞 Fix Form.Item set `label=""` will break the line align. [#43614](https://github.com/ant-design/ant-design/pull/43614)
- 🐞 Fix notification `placement` not being respected when passed via App component. [#43522](https://github.com/ant-design/ant-design/pull/43522) [@Rajil1213](https://github.com/Rajil1213)
- 🐞 Fix Pagination jumpy page size select when search in it. [#43556](https://github.com/ant-design/ant-design/pull/43556)
- 🐞 Fix Button disabled style is missing when use with the deprecated usage of `type="ghost"`. [#43558](https://github.com/ant-design/ant-design/pull/43558) [@kiner-tang](https://github.com/kiner-tang)
- 🐞 Fix Tag extra margin when there is only `icon` inside it. [#43518](https://github.com/ant-design/ant-design/pull/43518) [@Yuiai01](https://github.com/Yuiai01)
- 🐞 Fix ColorPicker that status style is missing inside Form.Item. [#42880](https://github.com/ant-design/ant-design/pull/42880) [@RedJue](https://github.com/RedJue)
- TypeScript
  - 🤖 Fix `SpaceContext` don't exported correctly. [#43501](https://github.com/ant-design/ant-design/pull/43501) [@VovkaGoodwin](https://github.com/VovkaGoodwin)
  - 🤖 Improve TS definitions for some components. [#43581](https://github.com/ant-design/ant-design/pull/43581) [#43545](https://github.com/ant-design/ant-design/pull/43545) [#43588](https://github.com/ant-design/ant-design/pull/43588) [#43610](https://github.com/ant-design/ant-design/pull/43610) [#43629](https://github.com/ant-design/ant-design/pull/43629). Thanks to [@thinkasany](https://github.com/thinkasany)、[@li-jia-nan](https://github.com/li-jia-nan) for the contributions.

---

- 💄 补全 Menu 主题定制 token。[#43573](https://github.com/ant-design/ant-design/pull/43573) [@li-jia-nan](https://github.com/li-jia-nan)
- 🐞 修复 QRCode 在 Next.js 13 中报错 `Can't resolve 'antd/lib/qr-code'` 的问题。[#43572](https://github.com/ant-design/ant-design/issues/43572)
- 🐞 修复 antd 不支持在 Next.js App Router 中使用的问题，查看[使用文档](/docs/react/use-with-next#使用-nextjs-的-app-router)。[#43576](https://github.com/ant-design/ant-design/pull/43576)
- 🐞 修复 App.useApp 方式调用 notification 组件时 `placement` 属性不生效的问题。[#43522](https://github.com/ant-design/ant-design/pull/43522) [@Rajil1213](https://github.com/Rajil1213)
- 🐞 修复 Checkbox 同时配置 `checked` 和 `indeterminate`` 时没有展示为 `indeterminate` 样式的问题。[#43626](https://github.com/ant-design/ant-design/pull/43626)
- 🐞 修复 Form.Item 设置 `label=""` 时垂直方向对齐偏移的问题。[#43614](https://github.com/ant-design/ant-design/pull/43614)
- 🐞 修复 Pagination 分页选择器弹层抖动的问题。[#43556](https://github.com/ant-design/ant-design/pull/43556)
- 🐞 修复 Button 幽灵按钮禁用状态丢失的问题。[#43558](https://github.com/ant-design/ant-design/pull/43558) [@kiner-tang](https://github.com/kiner-tang)
- 🐞 修复 Tag 仅传入 `icon` 时渲染多余间距的问题。[#43518](https://github.com/ant-design/ant-design/pull/43518) [@Yuiai01](https://github.com/Yuiai01)
- 🐞 修复 ColorPicker 不跟随表单校验状态改变 UI 的问题。[#42880](https://github.com/ant-design/ant-design/pull/42880) [@RedJue](https://github.com/RedJue)
- TypeScript
  - 🤖 修复 `SpaceContext` 没有正确导出的问题。[#43501](https://github.com/ant-design/ant-design/pull/43501) [@VovkaGoodwin](https://github.com/VovkaGoodwin)
  - 🤖 优化部分组件 TS 定义实现。[#43581](https://github.com/ant-design/ant-design/pull/43581) [#43545](https://github.com/ant-design/ant-design/pull/43545) [#43588](https://github.com/ant-design/ant-design/pull/43588) [#43610](https://github.com/ant-design/ant-design/pull/43610) [#43629](https://github.com/ant-design/ant-design/pull/43629)，感谢 [@thinkasany](https://github.com/thinkasany)、[@li-jia-nan](https://github.com/li-jia-nan) 的贡献。

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,19 @@ timeline: true
 
 ---
 
+## 5.7.1
+
+`2023-07-17`
+
+- 🐞 Fix that antd components usage in Next.js App Router. [#43573](https://github.com/ant-design/ant-design/pull/43573) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🐞 Fix Pagination jumpy page size select when search in it. [#43556](https://github.com/ant-design/ant-design/pull/43556)
+- 🐞 Fix Button disabled style of deprecated usage `type="ghost"`. [#43558](https://github.com/ant-design/ant-design/pull/43558) [@kiner-tang](https://github.com/kiner-tang)
```
Comment (kiner-tang) [inline CHANGELOG.en-US.md:24]:
```suggestion
- 🐞 Fix ghost button lose disabled state when use type='ghost'. [#43558](https://github.com/ant-design/ant-design/pull/43558) [@kiner-tang](https://github.com/kiner-tang)
```

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,20 @@ timeline: true
 
 ---
 
+## 5.7.1
+
+`2023-07-17`
```
Comment (li-jia-nan) [inline CHANGELOG.en-US.md:20]:
```suggestion
`2023-07-18`
```

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,20 @@ timeline: true
 
 ---
 
+## 5.7.1
+
+`2023-07-17`
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:20]:
```suggestion
`2023-07-18`
```

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,22 @@ timeline: true
...
 
+## 5.7.1
+
+`2023-07-18`
+
+- 💄 补全 Menu 主题定制 token。[#43573](https://github.com/ant-design/ant-design/pull/43573) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🐞 修复 QRCode 在 Next.js 13 中报错 `Can't resolve 'antd/lib/qr-code'` 的问题。[#43572](https://github.com/ant-design/ant-design/issues/43572)
+- 🐞 修复 antd 不支持在 Next.js App Router 中使用的问题，查看[使用文档](/docs/react/use-with-next#使用-nextjs-的-app-router)。[#43576](https://github.com/ant-design/ant-design/pull/43576)
+- 🐞 修复 App.useApp 方式调用 notification 组件时 `placement` 属性不生效的问题。[#43522](https://github.com/ant-design/ant-design/pull/43522) [@Rajil1213](https://github.com/Rajil1213)
+- 🐞 修复 Pagination 分页选择器弹层抖动的问题，类 Select 在页面高度变化时都不再抖动。[#43556](https://github.com/ant-design/ant-design/pull/43556)
```
Comment (xrkffgg) [inline CHANGELOG.zh-CN.md:26]: 类？

Comment (afc163) [inline CHANGELOG.zh-CN.md:26]:
```suggestion
- 🐞 修复 Pagination 分页选择器弹层抖动的问题。[#43556](https://github.com/ant-design/ant-design/pull/43556)
```

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -15,6 +15,22 @@ timeline: true
 
 ---
 
+## 5.7.1
+
+`2023-07-18`
```
Comment (li-jia-nan) [inline CHANGELOG.en-US.md:20]:
```suggestion
`2023-07-19`
```

File: CHANGELOG.zh-CN.md [hunk 6]
```diff
@@ -15,6 +15,22 @@ timeline: true
 
 ---
 
+## 5.7.1
+
+`2023-07-18`
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:20]:
```suggestion
`2023-07-19`
```

**Review by li-jia-nan — APPROVED** [review]

File: CHANGELOG.en-US.md [hunk 7]
```diff
@@ -15,6 +15,24 @@ timeline: true
 
 ---
 
+## 5.7.1
+
+`2023-07-19`
+
+- 💄 Migrate Component Token of Menu from 4.x less variables. [#43573](https://github.com/ant-design/ant-design/pull/43573) [@li-jia-nan](https://github.com/li-jia-nan)
```
Comment (MadCcc) [inline CHANGELOG.en-US.md:22]: 这条 changelog 和 PR 错了

**Pull request merged**